wire/net/test/connhub_internal_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	_ "perun.network/go-perun/backend/sim" // backend init
 	wallettest "perun.network/go-perun/wallet/test"
 	"perun.network/go-perun/wire"
+	_ "perun.network/go-perun/wire/protobuf" // wire serialzer init
```

Comment:
I don't like that the user now has to do these blank imports every time it uses perun. It's easy to forget. Is there a reminder in case you forget to set the serializer like this? What will happen then?

Reply:
I agree that this could be missed, in that case there will be a panic.

I think this (blank imports) is the way, we expect the user to initialize different backends (wallet, channel etc.,). So extended the same paradigm here.